examples/cpp/parabolic1D.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <math.h> 
+#include "mole.h"
+/**
+ * This example uses MOLE to solve the heat equation u_t-u_xx=0,u(x,0)=100x^2,u(-1,t)=100,u(1,t)=100
+ *the true solution should be approximately all 100s at cout;
+
+ */
+int main() { 
+int k=2;// Operators' order of accuracy
+double t0=0; //initial time
+double tf=1; //final time
+double a=0; //left boundary
+double b=1; //right boundary
+int m=2*k+1; //num of cells
+double dx=(b-a)/m;
+double dt=1.0/(ceil(3/(dx*dx)));
```

Comment:
Why was this chosen for the dt?

Reply:
Hello Jared, 
The reason i chose this for dt is so that if dx^2/3 isn't a multiple of t, we don't accidentally run the simulation longer than the final time. We need for the final time to be a multiple of t. And since I almost accidentally simulated this example over x in [-1,1] instead of x in [0,1]. With the value of m I picked, dx would have been 2/(9)[it's 2/9 because i almost did this with a 4th order laplacian and rk4 until the recent commit which makes it exactly like the matlab example], which means dx^2/3=(4/(243)), which means the final time t=1 isn't a multiple of dx^2/3 unlike the matlab example.